tensorflow/python/kernel_tests/bias_op_test.py

```diff
@@ -128,7 +128,13 @@ def _testGradient(self, np_input, bias, dtype, data_format, use_gpu):
           input_tensor, np_input.shape, output_tensor, np_input.shape)
       bias_jacob_t, bias_jacob_n = tf.test.compute_gradient(
           bias_tensor, bias.shape, output_tensor, np_input.shape)
-
+         
+      # Test gradient of BiasAddGrad
+      bias_add_grad = tf.gradients(tf.nn.l2_loss(output_tensor),
+                                   bias_tensor)[0]
+      grad_jacob_t, grad_jacob_n = tf.test.compute_gradient(
+          output_tensor, np_input.shape, bias_add_grad, bias.shape)
```

Comment:
I think we have a python3 issue here. Our nightlies are failing on this test:
http://ci.tensorflow.org/view/Nightly/job/nightly-matrix-linux-gpu/TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=NO_PIP,TF_BUILD_PYTHON_VERSION=PYTHON3,label=gpu-linux/247/consoleFull

I was also able to reproduce the problem locally using the following command:
$ bazel test tensorflow/python/kernel_tests:bias_op_test -c opt --config=cuda